build.sbt

```diff
@@ -30,7 +30,8 @@ val baseSettings = Seq(
   organization := "com.github.tototoshi",
   version := _version,
   scalaVersion := "2.12.7",
-  crossScalaVersions := Seq("2.11.12", "2.12.7"),
+  crossScalaVersions := Seq("2.11.12", "2.12.8", "2.13.0"),
+  fork in Test := true,
```

Comment:
This is just a workaround.
https://www.gitmemory.com/issue/sbt/sbt/4609/491662915